doc/libf3d/OPTIONS.md

```diff
@@ -17,6 +17,7 @@ See the exhaustive list below, but note that this may change in the future.
 
 Option|Type<br>Default<br>Trigger|Description|F3D option
 :---:|:---:|:---|:---:
+scene.animation.autoplay|bool<br>false<br>load|Automatically start animation.|\-\-animation-autoplay
```

Comment:
should be added to https://github.com/f3d-app/f3d/blob/master/doc/user/OPTIONS.md as well

Reply:
and to the changelog

Reply:
Got it! Should I add a bullet point for both "For F3D users" and "For libf3d users" since both are affected? Also just to confirm, this is for v2.2.1 in the changelog?

Reply:
> Should I add a bullet point for both "For F3D users" and "For libf3d users" since both are affected? 

Yes

> this is for v2.2.1 in the changelog?

Nope, please put it in `## Ongoing development` with the same logic.